src/main/java/dev/vasyl/proj/exception/CartEmptyException.java

```diff
@@ -0,0 +1,7 @@
+package dev.vasyl.proj.exception;
+
+public class CartEmptyException extends RuntimeException {
```

Comment:
better use a more general name, provide details in the message 
```suggestion
public class OrderProcessingException extends RuntimeException {
```